### PR TITLE
plugins/dracula.nvim: init

### PR DESCRIPTION
**Pull request by refaelsh**
This PR adds the dracula.nvim plugin.
Here the link to nixpkgs: https://search.nixos.org/packages?channel=24.05&show=vimPlugins.dracula-nvim&from=0&size=50&sort=relevance&type=packages&query=dracula-nvim.

Thanks in advance for reviewing :-)

### CODE REVIEW

**Review by MattSturgeon — APPROVED** [review]
Awesome, lgtm

**Comment by MattSturgeon** [comment]
@mergifyio queue

File: plugins/colorschemes/dracula-nvim.nix [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  defaultPackage = pkgs.vimPlugins.dracula-nvim;
+
+  maintainers = [ lib.nixvim.maintainers.refaelsh ];
+
+  settingsExample = {
+    settings = {
+      italic_comment = true;
+      colors.green = "#00FF00";
+    };
+  };
```
Comment (MattSturgeon) [inline plugins/colorschemes/dracula-nvim.nix:18]: Ah, I'm an idiot. This should've not had a nested `settings` within the example.

Also it seems I missed that there were unsquashed fixup commits.

**Comment by refaelsh** [comment]
> Ah, I'm an idiot. This should've not had a nested settings within the example.

Should I push the fix to `add-dracula-nvim3` or open a new PR?

> Also it seems I missed that there were unsquashed fixup commits.

Does not seem critical to me but:
1. I will take a not of it next time.
2. Its too late to do something about it, right?

**Comment by khaneliman** [comment]
> > Ah, I'm an idiot. This should've not had a nested settings within the example.
> 
> Should I push the fix to `add-dracula-nvim3` or open a new PR?
> 
Was handled in https://github.com/nix-community/nixvim/pull/2156
> > Also it seems I missed that there were unsquashed fixup commits.
> 
> Does not seem critical to me but:
> 
>     1. I will take a not of it next time.
> 
>     2. Its too late to do something about it, right?

Not without rewriting git history, which is highly discouraged on the trunk branch. So we just have to leave it be.

**Pull request merged**